schema/manifestlist_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/opencontainers/image-spec/schema"
+)
+
+func TestManifestList(t *testing.T) {
+	for i, tt := range []struct {
+		strict       bool
+		manifestList string
+		fail         bool
+	}{
+		// valid manifest list accepting customized media types
+		{
+			strict: false,
+			manifestList: `
+{
+  "schemaVersion": 2,
+  "mediaType": "application/customized+json",
```

Comment:
This [fails now][1] (since #404), because your example is not a manifest list.  I think you want to update this `mediaType` to `application/vnd.oci.image.manifest.list.v1+json`, but you should be able to leave the rest of this particular JSON blob alone.

[1]: https://travis-ci.org/opencontainers/image-spec/builds/176600776#L350